src/main/kotlin/org/jetbrains/kotlin/konan/target/Configurables.kt

```diff
@@ -86,6 +86,7 @@ interface AppleConfigurables : Configurables, ClangFlags {
     val osVersionMin get() = targetString("osVersionMin")!!
     val osVersionMinFlagLd get() = targetString("osVersionMinFlagLd")!!
     val testingUtilitiesDir get() = hostString("testingUtilitiesDir")
+    val absoluteTestingUtilitiesDir get() = absolute(testingUtilitiesDir)
```

Comment:
This probably should be overridden in `AppleConfigurables` to support using local Xcode instead of internal dependency.